### DISCUSSION TITLE
KHR_draco_mesh_compression

### DISCUSSION THREAD

**Discussion by goldenratio** [jnsmalm/pixi3d#102] [Q&A] [answered]
Hey,
is it possible to use KHR_draco_mesh_compression with pixi3d? https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_draco_mesh_compression/README.md

thanks for your help!

**Comment by jnsmalm** [comment, marked as answer]
Sorry, not supported.
